lib/engine/part/city.rb

```diff
@@ -7,25 +7,31 @@ module Engine
   module Part
     class City < RevenueCenter
       attr_accessor :reservations
-      attr_reader :tokens
+      attr_reader :tokens, :extra_tokens
```

Comment:
what is this?

Reply:
When a corp reaches its destination it has the option to place a token "outside" the city. So this is a list of tokens which are placed in the city, but do no occupy slots.

Reply:
i think there are other games that do this, so we shouldn't add this, just do 1870 specific behavior to add a slot

Reply:
I don't know of any. If I just add a slot, won't it cause a lot of problems with tile upgrades?

Reply:
let's add a slot and see if that's cleaner